### PR TITLE
drivers: added support for CC2420 [WIP]

### PR DESCRIPTION
**Pull request by haukepetersen**
though simple rx and tx works (to some extend), many bugs have to be fixed.

### CODE REVIEW

File: drivers/cc2420/include/cc2420_internal.h [hunk 1]
```diff
@@ -0,0 +1,250 @@
+/*
```
Comment (authmillenon) [inline drivers/cc2420/include/cc2420_internal.h]: Any reason why you need the `include` directory here? Putting it just into `drivers/cc2420/` would simplify the Makefiles a lot, I think.

File: drivers/cc2420/include/cc2420_params.h [hunk 2]
```diff
@@ -0,0 +1,88 @@
+/*
```
Comment (authmillenon) [inline drivers/cc2420/include/cc2420_params.h]: Should go into `drivers/include` IMHO.

Comment (haukepetersen) [inline drivers/cc2420/include/cc2420_params.h]: hm, this is the place we put it for all (network) devices so far... If we move it, we should IMHO move it for all of them, so separate issue?!

File: drivers/cc2420/include/cc2420_internal.h (hunk 1 above)
Comment (haukepetersen) [inline drivers/cc2420/include/cc2420_internal.h]: no particular reason, just followed all the other devices...

File: drivers/cc2420/include/cc2420_params.h (hunk 2 above)
Comment (authmillenon) [inline drivers/cc2420/include/cc2420_params.h]: kk

**Comment by kYc0o** [comment]
Closed by #5591

**Pull request closed without merging**